frontend/src/components/messstelle/MessstelleForm.vue

```diff
@@ -0,0 +1,275 @@
+                        md="12"
+                    >
+                        <v-textarea
+                            v-model="editMessstelle.bemerkung"
+                            label="Bemerkung"
+                            outlined
+                            readonly
+                            dense
+                            rows="1"
+                            row-height="10"
+                        ></v-textarea>
+                    </v-col>
+                </v-row>
+                <v-row dense>
+                    <v-col
+                        cols="12"
+                        md="12"
+                    >
+                        <v-combobox
+                            v-model="editMessstelle.customSuchwoerter"
+                            multiple
+                            label="Suchwörter"
+                            outlined
+                            dense
+                            small-chips
+                            deletable-chips
+                            class="tag-input"
+                            :search-input.sync="newSuchwort"
+                            append-icon="mdi-plus"
+                            @click:append="addSuchwortToList"
+                            @keyup.enter="addSuchwortToList"
+                            @blur="addSuchwortToList"
+                        >
+                        </v-combobox>
+                    </v-col>
+                </v-row>
+                <v-row dense>
+                    <v-col
+                        cols="12"
+                        md="12"
+                    >
+                        <v-textarea
+                            v-model="editMessstelle.kommentar"
+                            label="Kommentar"
+                            outlined
+                            dense
+                            rows="2"
+                            row-height="10"
+                            counter="255"
+                            maxlength="255"
+                        ></v-textarea>
+                    </v-col>
+                </v-row>
+            </v-form>
+        </v-card-text>
+    </v-sheet>
+</template>
+
+<script setup lang="ts">
+import { computed, ComputedRef, ref, Ref } from "vue";
+/* eslint-disable no-unused-vars */
+import MessstelleEditDTO from "@/domain/dto/messstelle/MessstelleEditDTO";
+import LhmTextField from "@/components/common/LhmTextField.vue";
+
+/* eslint-enable no-unused-vars */
+
+interface Props {
+    height: string;
+    value: MessstelleEditDTO;
+}
+
+const props = defineProps<Props>();
+
+const emits = defineEmits<{
+    (e: "input", v: MessstelleEditDTO): void;
+}>();
+
+const editMessstelle = computed({
+    get: () => props.value,
+    set: (v) => emits("input", v),
+});
+
+const detektierteFahrzeuge: ComputedRef<string> = computed(() => {
+    let result = "";
+    if (
+        editMessstelle.value.messquerschnitte &&
+        editMessstelle.value.messquerschnitte.length > 0
```

Comment:
da in der darauffolgenden Methode das gleiche verwenden wird, könnte man das in ne extra funktion packen und dann nen ternary drauß machen. also:

```
function detektierteFahrzeuge() {
    return doesMessquerschnitteExist ?  editMessstelle.value.messquerschnitte[0].detektierteVerkehrsarten : "";
}

function doesMessquerschnitteExist():boolean {
   return  editMessstelle.value.messquerschnitte && editMessstelle.value.messquerschnitte.length > 0;
}
```

Reply:
Umgestellt auf isEmpty von lodash